test/jasmine/tests/gl3daxes_test.js

```diff
@@ -64,14 +64,17 @@ describe('Test Gl3dAxes', function() {
                 var keys = Object.keys(validObject);
                 for(var i = 0; i < keys.length; i++) {
                     var k = keys[i];
-                    if(validObject[k] !== testObject[k]) return false;
+                    expect(validObject[k]).toBe(testObject[k]);
+                    // if(validObject[k] !== testObject[k]) return false;
                 }
                 return true;
             }
 
             supplyLayoutDefaults(layoutIn, layoutOut, options);
             ['xaxis', 'yaxis', 'zaxis'].forEach(function(axis) {
-                expect(checkKeys(expected[axis], layoutOut[axis])).toBe(true);
+                // expect(
+                checkKeys(expected[axis], layoutOut[axis]);
+                // ).toBe(true);
```

Comment:
Ah I see. I thought the whole block was commented. My bad. Ignore :arrow_double_up: .